src/server/graphql/mutations/__tests__/removeTeamMember.test.js

```diff
@@ -0,0 +1,88 @@
+import MockDate from 'mockdate';
+import mockAuthToken from 'server/__tests__/setup/mockAuthToken';
+import {__now} from 'server/__tests__/setup/mockTimes';
+import removeTeamMember from 'server/graphql/mutations/removeTeamMember';
+import MockDB from 'server/__tests__/setup/MockDB';
+import MockPubSub from 'server/__mocks__/MockPubSub';
+import fetchAndSerialize from 'server/__tests__/utils/fetchAndSerialize';
+import getRethink from 'server/database/rethinkDriver';
+import {auth0ManagementClient} from 'server/utils/auth0Helpers';
+import DynamicSerializer from 'dynamic-serializer';
+import * as getPubSub from 'server/utils/getPubSub';
+import expectAsyncToThrow from 'server/__tests__/utils/expectAsyncToThrow';
+
+MockDate.set(__now);
+console.error = jest.fn();
+
+describe('removeTeamMember', () => {
+  test('promotes another member if the person removed was the lead', async () => {
+
+  });
+
+  test('removes the teamMember, reassigns active projects to the lead', async () => {
```

Comment:
This is such a nitpick, but I like to use `it`, since "it removes the teamMember..." makes just a little more sense to my reptilian brain than "test removes the teamMember..."

Reply:
oh man, me too! either relay used to not allow `it`, or i didn't know it could until half the tests were already written. it's one of those things where i want it to be consistent across the codebase, but i never took the time to change everything from `test` to `it`.